packages/headless/src/controllers/commerce/core/facets/headless-core-commerce-facet.ts

```diff
@@ -281,12 +286,20 @@ export function buildCoreCommerceFacet<
 
     get state() {
       const response = getResponse();
+      const canShowMoreValues = response?.moreValuesAvailable ?? false;
+
+      const canShowLessValues = (() => {
+        const request = getRequest();
```

Comment:
Can we do an early return if there's no request, simplifying the rest of the function since we can remove all the `?` operator/check for undefined sprinkled throughout.

Reply:
Addressed this in https://github.com/coveo/ui-kit/pull/3495/commits/6dd9a472cc7bad15df5dcaebce244f2e21ae37ea